index.html

```diff
@@ -253,11 +253,12 @@ <h3>Ecosystem Overview</h3>
 <a>verifiable credentials</a> for processing.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
-        <dt><a>identifier registry</a></dt>
+        <dt><a>Verifiable data repository</a></dt>
         <dd>
-A role a system may perform by mediating the creation and verification of
-<a>subject</a> identifiers. Examples of identifier registries include
-corporate employee databases, government ID databases, and distributed ledgers.
+Mediates the creation and verification of <a>issuer</a> identifiers, keys and
```

Comment:
The registry is not only for issuers, it's for subjects in general of which issuers are a subtype.

Reply:
Fixed